model/src/main/kotlin/config/PackageConfiguration.kt

```diff
@@ -98,17 +98,19 @@ data class VcsMatcher(
     val url: String,
 
     /**
-     * The [revision] to match for equality against [RepositoryProvenance.resolvedRevision].
+     * The [revision] to match for equality against [RepositoryProvenance.resolvedRevision], or null to match any
+     * revision.
      */
-    val revision: String
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    val revision: String? = null
```

Comment:
Please also update `docs/config-file-package-configuration-yml.md`, maybe just add "(optional)" before the one mention of the revision property.